narwhals/functions.py

```diff
@@ -21,14 +22,14 @@
 def concat(
     items: Iterable[FrameT],
     *,
-    how: Literal["horizontal", "vertical"] = "vertical",
+    how: Literal["horizontal", "vertical"] | Any = "vertical",
```

Comment:
why `Any`?

Reply:
Because how else would it enter into the if statement? If it is typed as `Literal["vertical", "horizontal"]` there should be no code assuming is something else like the if block at the beginning